backend/src/v2/Makefile

```diff
@@ -18,7 +18,7 @@ SHELL = /bin/bash
 # create an .env file to override dev vars locally
 -include .env
 DEV_IMAGE_PREFIX?=gcr.io/ml-pipeline-test/dev/kfp-
-REPO_ROOT=..
+REPO_ROOT=../../..
```

Comment:
revert

Reply:
I think this needs to be kept if we want to run the Makefile successfully. Probably due to the v2 folder used to be under the root folder.

Reply:
I see. You're right.

Reply:
I see. You're right.